README.md

```diff
@@ -0,0 +1,42 @@
+# Blogger
+
+Copyright 2020, The Basebandit
+
+## Licensing
```

Comment:
I would put your terms in a licence file named `LICENSE` (and I would use the MIT licence, unless there's a good reason not to).

Reply:
Thank you I will do that..sounds much better having the license file separately.